docs/ja/about/security.md

```diff
@@ -4,32 +4,34 @@ title: セキュリティ
 category: About
 menu: menu_ja
 toc:
-    - title: セキュリティ
-      url: "#セキュリティ"
-    - title: 脆弱性の報告
-      url: "#脆弱性の報告"
-    - title: チーム
-      url: "#チーム"
-    - title: 過去のセキュリティ問題
-      url: "#過去のセキュリティ問題"
+  - title: セキュリティ
+    url: "#セキュリティ"
+  - title: 脆弱性の報告
+    url: "#脆弱性の報告"
+  - title: チーム
+    url: "#チーム"
+  - title: 過去のセキュリティ問題
+    url: "#過去のセキュリティ問題"
 ---
+
 # セキュリティ
+
 私たちはセキュリティを非常に重視していますが複雑なソフトウェアプロジェクトには、脆弱性がつきものです。
 私たちはプロジェクトの中核となるメンテナと協力して、セキュリティ上の脆弱性を評価し、タイムリーに対処しています。
 
-
 ## 脆弱性の報告
-セキュリティ上の問題は、私たちのプライベートメーリングリスト<screwdriver-oss-security@yahooinc.com>に報告してください。
-セキュリティ以外の問題については、[github](./support#github)のIssue、または[slack](./support#slack)を使用してください。
 
+セキュリティ上の問題は、私たちのプライベートメーリングリスト<screwdriver-oss-security@yahooinc.com>に報告してください。
+セキュリティ以外の問題については、[github](./support#github)の Issue、または[slack](./support#slack)を使用してください。
 
 ## チーム
-Screwriverセキュリティチームは、Screwdriverの[コアメンテナ](https://github.com/orgs/screwdriver-cd/teams/security)で構成されています。
-私たちの目標は、Screwdriverのセキュリティを向上させ、クラスター管理者にScrewdriverを構築するクラスターのセキュリティを確保するために必要なツールや情報を提供することです。
+
+Screwriver セキュリティチームは、Screwdriver の[コアメンテナ](https://github.com/orgs/screwdriver-cd/teams/security)で構成されています。
+私たちの目標は、Screwdriver のセキュリティを向上させ、クラスター管理者に Screwdriver を構築するクラスターのセキュリティを確保するために必要なツールや情報を提供することです。
 
 ### チームへの参加
-ご興味のある方は、セキュリティチーム（<screwdriver-oss-security@verizonmedia.com>）までご連絡ください。会員になれるのは、Screwdriverプロジェクトに貢献した経験のある方に限られ、セキュリティチームの[現在のメンバー](https://github.com/orgs/screwdriver-cd/teams/security)による承認が必要になります。
 
+ご興味のある方は、セキュリティチーム（<screwdriver-oss-security@yahooinc.com>）までご連絡ください。会員になれるのは、Screwdriver プロジェクトに貢献した経験のある方に限られ、セキュリティチームの[現在のメンバー](https://github.com/orgs/screwdriver-cd/teams/security)による承認が必要になります。
```

Comment:
I am assuming that no Japanese text was modified here. If not, then we need to get this reviewed by YJ

Reply:
yeah no, probably just due to the editor autosaving.